CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 1.9.0 – 2022-01-18
+### Changed
+- Dependency updates (esp. axios v0.25.0)
```

Comment:
I would mark that as security, no?

Reply:
I added a second section `Fixed` with a note to the fixed vulnerability.